packages/runtime/container-runtime/README.md

```diff
@@ -51,6 +51,54 @@ issue](https://github.com/microsoft/FluidFramework/wiki/Submitting-Bugs-and-Feat
 
 Thank you!
 
+## Data Virtualization For DataStores
+
+This is the ability of container to delay load datastores at any point in time by asking for the datastore snapshot at
+anytime, not caring about from where it is coming and then using that to load that datastore and return it to the app.
+
+### How Current System Works
+
+Fluid (through its APIs) does expose some levels of data virtualization. But under covers, there is almost none. The
+whole content of the file is downloaded in one go. Due to limitation of data virtualization, FF holds ALL blobs in
+snapshot as those might be required in the future. Any delayed loading (through FF APIs) results in loading state of
+DDSs at a sequence number of snapshot we booted from, and application of all the ops up till current sequence number.
+While application may choose not to load some data stores immediately on boot (and realize some saving in time and
+memory by not allocating appropriate app state for such datastores), FF still pays the costs for such content.
+It also continues to pay the cost for all such content indefinitely, even if those datastores were loaded (this could
+theoretically be optimized, but now there is no good way for driver to learn what blobs are safe to discard due to
+limitations in interface and lack of appropriate semantics at driver API layer).
```

Comment:
I could rename the heading and some text and put it in section like:
"Motivation for Data Virtualization" and mention what used to happen before Dv. It would be good for the reader to know about it.

Reply:
Maybe later when we are already in the DV world for some time, then we can remove it later on as I would think we would keep updating this readme as we implement more features and capabilities based on DV.